


function main()
{
	console.log("In main function")
}


function error()
{
	let y = 0
}

main()
console.log(error())